airbyte-integrations/connectors/source-google-sheets/pyproject.toml

```diff
@@ -12,18 +12,15 @@ readme = "README.md"
 documentation = "https://docs.airbyte.com/integrations/sources/google-sheets"
 homepage = "https://airbyte.com"
 repository = "https://github.com/airbytehq/airbyte"
-packages = [ { include = "source_google_sheets"}]
+[[tool.poetry.packages]]
```

Comment:
I changed this line because it didn't match the pattern from other connectors. My understanding is that it's essentially just a different syntax, but I figure it's worth being sticklers for conformity with the poetry files.